releases/release-1.13/patch_releases.md

```diff
@@ -0,0 +1,37 @@
+# 1.13 Patch Releases
+
+Schedule and team contact information for Kubernetes 1.13 patch releases.
+
+For general information about Kubernetes release cycle, see https://github.com/kubernetes/community/blob/master/contributors/devel/release.md
+
+# 1.13 Patch Release Management Team
+
+To discuss a cherry-pick, schedule, or anything else related to 1.13 patch releases,
+contact any member of the team. You can just tag us on #sig-release Slack channel
+or message directly. Please give us a business day to respond - we may be in a different timezone!
+
+Cherry-picks are monitored on regular basis, so it's likely you'll hear
+from us on PRs to release-1.13 branch by the time the next patch release is due.
+
+| **Github username** | **Slack username** | **Timezone** |
+| --- | --- | --- |
+| @aleksandra-malinowska | @aleksandram | CET |
+| @tpepper | @tpepper | PST |
+
+# Timeline
+
+Next patch release is **1.13.2**.
+
+Cherry-picks deadline is the end of day by which all pending PRs to release-1.13
+branch have to merge to be included in the patch release. Any PRs left pending past
+this point will either wait for the next patch release, or may in rare cases
+delay current release.
+
+Target date is the date on which the patch release is cut. Must be at least 2
+days after cherry-picks deadline to allow all release-blocking tests to run.
+
+| **Patch Release** | **Cherry-picks deadline** | **Target date** | **Owner** |
+| --- | --- | --- | --- |
+| 1.13.2 | TBA | 2019-01-10 | @tpepper |
```

Comment:
I'll leave it to you to decide what's appropriate, but I would personally prefer to see a longer forecast.  Something like a suggested cadence, or the next N potential dates, etc.